tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -211,8 +211,8 @@ def check_roslaunch(f):
     
     # load all node defs
     nodes = []
-    for filename, rldeps in file_deps.items():
-        nodes.extend(rldeps.nodes)
+    for node in rl_config.nodes():
```

Comment:
This result in the following error for me:

    TypeError: 'list' object is not callable